docs/source/plugin/bot.rst

```diff
@@ -220,32 +220,69 @@ Channels & users
 
 Knowing how to talk is good for a bot, but you may be wondering what the bot
 knows about the channels and their users. For that, you can use the bot's
-:attr:`~sopel.bot.Sopel.channels` attribute. For example, to list all channels
-the bot is in::
+:attr:`~sopel.bot.Sopel.channels` attribute.
+
+For example, to list all channels the bot is in::
 
     for name, channel in bot.channels.items():
         # do something with the name and the channel
 
+.. note::
+
+    Sopel doesn't know about channel it didn't join first, and it forgets
+    everything about channel it leaves.
```

Comment:
🚨 Nemesi**s** spotted, cap'n! 🚨

Two nemese**s**, actually, but I took the other one in a different direction.

```suggestion
    Sopel doesn't know about channels it didn't join first, and it forgets
    everything about a channel when it leaves.
```

Reply:
Alright, I forgot these ones. However, I didn't forget about "it forgets". Improvement!